rasa/cli/test.py

```diff
@@ -53,7 +54,7 @@ def add_subparser(
 
     test_core_parser.set_defaults(func=test_core)
     test_nlu_parser.set_defaults(func=test_nlu)
-    test_parser.set_defaults(func=test)
+    test_parser.set_defaults(func=test, e2e=True, stories=DEFAULT_E2E_TESTS_PATH)
```

Comment:
Mhm, shouldn't we exclude the `e2e` parameter for `rasa test` because you can't change it anyway?